runtime/compiler/env/j9method.cpp

```diff
@@ -744,10 +744,12 @@ static const char * const excludeArray[] = {
    "java/lang/reflect/AccessibleObject.invokeF(Ljava/lang/Object;[Ljava/lang/Object;)F",
    "java/lang/reflect/AccessibleObject.invokeD(Ljava/lang/Object;[Ljava/lang/Object;)D",
    "java/lang/reflect/AccessibleObject.invokeL(Ljava/lang/Object;[Ljava/lang/Object;)Ljava/lang/Object;",
+#if (24 < JAVA_SPEC_VERSION)
```

Comment:
If I'm understanding [`isCompilable`](https://github.com/eclipse-openj9/openj9/blob/a8dcdd8a8d2cd3d330a43e9d50c31b496679cbf4/runtime/compiler/env/j9method.cpp#L773-L775) correctly, any method whose signature is found in `excludeArray` will not be compiled.  So if we want these methods to be compiled for JDK 24 or later, but not for JDK 23 or earlier, I think the condition should be:

```
#if (JAVA_SPEC_VERSION < 24)
```

Sorry if I'm misunderstanding what this is doing!

Reply:
Sorry - that suggestion should have omitted the parentheses, as @keithc-ca pointed out.

Reply:
That is consistent with how I read `isCompilable()`. I think my confusion stems from the use of the word "include"; those methods are included in a table of exclusions - maybe the commit message should avoid the terms "include" and "exclude" and better express the goal of this change: to allow the JIT compiler to consider compiling those methods for Java 24+.

So the code should read:
```c
#if JAVA_SPEC_VERSION < 24
```